sdk/textanalytics/Azure.AI.TextAnalytics/src/AspectSentiment.cs

```diff
@@ -57,5 +57,11 @@ internal AspectSentiment(SentenceAspect sentenceAspect)
         /// Gets the starting position (in UTF-16 code units) for the aspect text.
         /// </summary>
         public int Offset { get; }
+
+        /// <inheritdoc/>
+        public override string ToString()
+        {
+            return $"Text: {Text}, Sentiment: {Sentiment}, Confidence Scores: [{ConfidenceScores}]";
```

Comment:
Nested objects. What is the format? 
- `nameof(property): {{ConfidenceScores}}`
- `nameof(property): [{ConfidenceScores}]`
- ...